pom.xml

```diff
@@ -62,14 +62,14 @@
     </build>
     <distributionManagement>
         <repository>
```

Comment:
This replacement is incorrect, because it replaces the new repository with th old.